xbmc/cores/VideoPlayer/DVDCodecs/Audio/DVDAudioCodecFFmpeg.cpp

```diff
@@ -184,7 +184,7 @@ void CDVDAudioCodecFFmpeg::GetData(DVDAudioFrame &frame)
 {
   frame.nb_frames = 0;
 
-  uint8_t* data[16];
+  uint8_t* data[16]{};
```

Comment:
This seems odd as it's used by the next line.

Reply:
You could use `std::array`

Reply:
RE std:array, not really looking to change behaviour at this point.
Same reasoning as the other one, a couple brackets removes a static analysis warning.